webapp/channels/src/components/readout/readout.test.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {render, screen, act} from '@testing-library/react';
+import React from 'react';
+import {Provider} from 'react-redux';
+import configureStore from 'redux-mock-store';
+
+import {ActionTypes} from 'utils/constants';
+
+import Readout from './readout';
+
+describe('Readout', () => {
+    const mockStore = configureStore([]);
+
+    it('should render message and clear it after timeout', async () => {
+        jest.useFakeTimers();
+
+        const store = mockStore({
+            views: {
+                readout: {
+                    message: 'Test message',
+                },
+            },
+        });
+
+        render(
+            <Provider store={store}>
+                <Readout/>
+            </Provider>,
+        );
```

Comment:
We have `renderWith` util functions to render with the store. Any reason not to use those?

Reply:
Took a look and I think there's no reason to use them since they provide functionality that the test doesn't need. We don't need to properly mock out the store, just ensure that the timeout works.